### ISSUE TITLE
Switch tests that require external services (ssh/minio) to integration tests

### ISSUE THREAD

**Issue by detiber** [kubicorn/kubicorn#569]
Tests accessing external services should be run as separate integration tests, rather than as unit tests.

It would also be beneficial to not require pre-configured and running external containers to provide these services, but rather to spin up services as part of the test or test harness setup.

**Comment by kris-nova** [comment]
I agree - we should move all integrations tests out of the main repo and not as unit tests
